esp8266-fastled-webserver/TwinkleFOX.cpp

```diff
@@ -59,6 +59,7 @@
 //  smoothly at over 50 updates per seond.
 //
 //  -Mark Kriegsman, December 2015
+#include "common.h"
```

Comment:
SIMPLE FILE REVIEW

File existed as TwinkleFOX.h in many of the active branches.

* Main change was to use a common base function for both
the fibonacci and non-fibonacci variants.
* Changed to use `NUM_PIXELS` instead of `NUM_LEDS`
* Added `FL_ALIGN_PROGMEM` to ensure `FL_PROGMEM` structures would be properly aligned on all architectures.